rc_dev

```diff
@@ -14,3 +14,4 @@ export WMS_URL=//wms-bgdi.dev.bgdi.ch
 export WMTS_URL=//tod.dev.bgdi.ch
 export TERRAIN_URL=//3d.dev.bgdi.ch
 export VECTORTILES_URL=//vectortiles{s}.dev.bgdi.ch/3d-tiles
+export S3_CONFIG_BASE=swisstopo-internal-filesharing.s3.amazonaws.com/config/mf-geoadmin3/dev
```

Comment:
@procrastinatio , `swisstopo-internal-filesharing.s3.amazonaws.com` is a bad candidate to store client configuration. Or do I get it wrong? Is the application using it directly or is it used by chsdi3?
There is no cloudfront in front of that bucket, and probably routing also is not working outside swisstopo (unverified). For the client, I would statify the configuration in a bucket with cloudfront. Might be a dedicated one or the one of map.geo.admin.ch.

Reply:
It is still WIP. And with `mf-geoadmin3.dev.bgdi.ch`files are served directly by `mf-chsdi3.dev.bgdi.ch`as now.